Engine/TestPlanExecution.cs

```diff
@@ -701,8 +720,10 @@ public TestPlanRun Execute(IEnumerable<IResultListener> resultListeners, IEnumer
         }
 
         TestPlanRun currentExecutionState = null;
+        internal const string breakConditionParameterName = "BreakIssuedFrom";
```

Comment:
Lets move these to a class called SpecialParameterNames. And make them public.

Reply:
Great idea. I have added it.